manof/image.py

```diff
@@ -127,6 +128,10 @@ def run(self):
             for k, v in self.labels.items():
                 command += '--label {0}={1} '.format(k, v)
 
+        command += '--label {0}={1} '.format(
+            'manof.commandSHA', "<Replace:manof.commandSHA>"
```

Comment:
Keep the `manof.commandSHA` and `<Replace:manof.commandSHA>` as a constants somewhere and use them in all the places you need.